web_modules/Wrapper/index.css

```diff
@@ -16,18 +17,18 @@
 :global .markdownIt-Anchor {
   position: absolute;
   left: -1.5rem;
-  opacity: .1;
-  transition: opacity .2s;
+  opacity: 0.1;
+  transition: opacity 0.2s;
 }
 
-  :global h1:hover .markdownIt-Anchor,
-  :global h2:hover .markdownIt-Anchor,
-  :global h3:hover .markdownIt-Anchor,
-  :global h4:hover .markdownIt-Anchor,
-  :global h5:hover .markdownIt-Anchor,
-  :global h6:hover .markdownIt-Anchor {
-    opacity: 1;
-  }
+:global h1:hover .markdownIt-Anchor,
+:global h2:hover .markdownIt-Anchor,
+:global h3:hover .markdownIt-Anchor,
+:global h4:hover .markdownIt-Anchor,
+:global h5:hover .markdownIt-Anchor,
+:global h6:hover .markdownIt-Anchor {
+  opacity: 1;
+}
```

Comment:
@jeddy3 is there a way to indent based on the "nesting" of the selector? (to keep this?)